onnx/test/shape_inference_test.py

```diff
@@ -173,6 +194,34 @@ def _compare_value_infos(
                 "Unrecognized value info type in _compare_value_infos: ", str(vi_type)
             )
 
+    def _test_an_op(self, op_name, test_function_name) -> None:
+        """Given an operator name and test_function_name, run tests for op versions > 5.
```

Comment:
Add an explanation on why >5 for clarity?

Reply:
Added following sentences:
```
        However, there is a creation issue for Reshape Operator under version 5, which
        probably caused by the mismatch creation arguments in _make_graph.
        Therefore, all test cases now only run when version > 5.
```